README.md

```diff
@@ -1,2 +1,3 @@
 # commercetools-subscriptions
+
 Cron job based service which manages subscriptions, reminders and triggers subscription orders based on the schedule
```

Comment:
```suggestion
Cron job based service which manages subscriptions, reminders and triggers subscription orders based on a schedule
```

Reply:
I guess here it doesn't matter. I'm not sure if your suggestion is correct, so I'll leave it as it is.